sympy/diffgeom/diffgeom.py

```diff
@@ -499,17 +572,70 @@ def coord_tuple_transform_to(self, to_sys, coords):
 
     def jacobian(self, sys, coordinates=None):
         """
-        Return the jacobian matrix of a transformation.
+        Return the jacobian matrix of a transformation on given coordinates.
+        If coordinates are not given, coordinate symbols of *self* are used.
+
+        Parameters
+        ==========
+
+        sys : CoordSystem
+
+        coordinates : Any iterable, optional.
+
+        Returns
+        =======
+
+        sympy.ImmutableDenseMatrix
+
+        Examples
+        ========
+
+        >>> from sympy.diffgeom.rn import R2_r, R2_p
+        >>> R2_p.jacobian(R2_r)
+        Matrix([
+        [cos(theta), -rho*sin(theta)],
+        [sin(theta),  rho*cos(theta)]])
+        >>> R2_p.jacobian(R2_r, [1, 0])
+        Matrix([
+        [1, 0],
+        [0, 1]])
+
         """
         result = self.transform(sys).jacobian(self.symbols)
         if coordinates is not None:
             result = result.subs(list(zip(self.symbols, coordinates)))
         return result
-    jacobian_matrix = jacobian
```

Comment:
No, I don't think that you should remove this line completely because it was there in the previous version.
But I would rather remove `jacobian_det` below if that is also accidental

Reply:
Oops. This was a mistake.

BTW I think the real names for two methods should be `jacobian_mat()` and `jacobian_det()` for clarity, since the term "jacobian" is ambiguous. (https://mathworld.wolfram.com/Jacobian.html)

Reply:
I don't want this topic to be a blocker for this PR so I'll open a separate issue for the naming of methods.